makefile

```diff
@@ -0,0 +1,25 @@
+#
+# Makefile
+#
+
+.PHONY: help build
+.DEFAULT_GOAL := help
+
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+# ----------------------------------------------------------------------------------------------------------------
+
+install: ## Installs all dependencies
+	go mod download
+
+build: ## Builds the binary
+	env GOOS=linux GOARCH=amd64 go build -o ./build/linux/amd64/dockware-cli .
```

Comment:
you could use also goreleaser 🤔

Reply:
I though about that, but as long as boxblinkracer is just releasing a binary I'd say the task is too simple to warrant another dependency, and make is pretty much universally available.

Reply:
hehe, totally open for that,
at the moment theres just a small automation in the background to release new binaries on the website (didnt have more time)
but we can totally change and improve it :)